pom.xml

```diff
@@ -71,14 +71,16 @@
         <dependency>
             <groupId>io.airlift</groupId>
             <artifactId>testing</artifactId>
-            <version>218</version>
+            <!-- 219 is the last version targeted for JDK 11 -->
```

Comment:
@dain @martint should we update units to JDK 17?

Reply:
It's used in JDBC and client

Reply:
Already a part of https://github.com/airlift/units/commit/9336b77ca379c12986a2b7051c280c7b25a013b2